e2e/src/app.e2e-spec.ts

```diff
@@ -27,10 +27,6 @@ describe('Algorea Frontend', () => {
     });
 
     it('should have a working collapse button', async () => {
-      await expect(page.getLeftElement().getAttribute('class')).toMatch('expanded');
-      await expect(page.getTopBarElement().getAttribute('class')).toMatch('expanded');
-      await expect(page.getRightElement().getAttribute('class')).toMatch('expanded');
-
```

Comment:
Later in the tests we expect the class of those element to match `'collapsed'`
So I guess here we would expect the class of the same elements **not** to match `'collapsed'` instead of removing the tests

Reply:
Fixed